geomesa-tools/bin/common-functions.sh

```diff
@@ -279,6 +279,49 @@ function registerAutocomplete() {
   fi
 }
 
+function installScala() {
+  # Download scala to %%gmtools.dist.name%%_HOME}/dist/
+  sourceURL="https://downloads.lightbend.com/scala/%%scala.version%%/scala-%%scala.version%%.tgz"
+  outputFile="${%%gmtools.dist.name%%_HOME}/dist/scala-%%scala.version%%.tgz"
+  wget $sourceURL -O $outputFile
```

Comment:
prefer `curl` over `wget`, as it's more widely installed. you can just call the 'downloadUrls' function, which will take care of it for you. it has a confirmation prompt, so you'll want to check that the file was actually downloaded before continuing. the logic below will need to be tweaked slightly, so you don't have double confirmations.

Reply:
+1

Reply:
Good Idea, I didn't notice that function.